scripts/bump_version.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+const semver = require('semver');
+const exec = require('./utils/exec');
+const packagesIterator = require('./utils/packagesIterator');
+const rootPackageJson = require('../package.json');
+
+const convertReleaseToPrerelease = (version) => {
+  const bumpedVersion = semver.inc(version, 'minor');
+
+  return `${semver.major(bumpedVersion)}.${semver.minor(bumpedVersion)}.0-dev.1`;
+};
+
+(async () => {
```

Comment:
Node 16 support await on module level. Isn't it?

Reply:
Nope

Reply:
Right, my bad.